faq/manage-cluster-faq.md

```diff
@@ -35,16 +37,20 @@ By default, TiDB/PD/TiKV outputs standard error in the logs. If a log file is sp
 
 ### How to safely stop TiDB?
 
-Kill all the services using `kill` directly. The components of TiDB will do `graceful shutdown`.
+- A load balancer is running (recommended): Stop the load balancer and then run the kill command. Then TiDB waits for a period specified by [graceful-wait-before-shutdown](/tidb-configuration-file.md#graceful-wait-before-shutdown-new-in-v50) until all sessions are terminated. Then TiDB stops running.
+
+- No load balancer is running: Run the kill command. The TiDB components are gracefully stopped.
 
 ### Can `kill` be executed in TiDB?
 
-- You can `kill` DML statements. First use `show processlist` to find the ID corresponding with the session, and then run `kill tidb [session id]`.
-- You can `kill` DDL statements. First use `admin show ddl jobs` to find the ID of the DDL job you need to kill, and then run `admin cancel ddl jobs 'job_id' [, 'job_id'] ...`. For more details, see the [`ADMIN` statement](/sql-statements/sql-statement-admin.md).
+[Kill](/sql-statements/sql-statement-kill.md) commands are disabled by default. To run a kill command, set `compatible-kill-query = true` or enable Global Kill (experimental).
+
+- Kil DML statements: First use `show processlist` to find the ID corresponding with the session, and then run `kill tidb [session id]`.
+- Kill DDL statements: First use `admin show ddl jobs` to find the ID of the DDL job you need to kill, and then run `admin cancel ddl jobs 'job_id' [, 'job_id'] ...`. For more details, see the [`ADMIN` statement](/sql-statements/sql-statement-admin.md).
 
 ### Does TiDB support session timeout?
 
-TiDB does not currently support session timeout at the database level. At present, if you want to achieve timeout, when there is no LB (Load Balancing), you need to record the ID of the initiated Session on the application side. You can customize the timeout through the application. After timeout, you need to go to the node that initiated the Query Use `kill tidb [session id]` to kill SQL. It is currently recommended to use an application program to achieve session timeout. When the timeout period is reached, the application layer will throw an exception and continue to execute subsequent program segments.
+TiDB currently supports two timeouts, [`wait_timeout`](/system-variables.md#wait_timeout) and [`interactive_timeout`](/system-variables.md#interactive_timeout).
```

Comment:
We could mention the max_execution_time query timeout here, but I don't think we should